frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -110,8 +185,79 @@ const filters = [
   },
 ];
 
-const NodesPage = (props) => (
-  <ListPage {...props} ListComponent={NodesTable} rowFilters={filters} />
-);
+const fetchNodeMetrics = (): Promise<NodeMetrics> => {
+  const metrics = [
+    {
+      key: 'freeMemory',
+      query: 'node_memory_MemAvailable_bytes',
+    },
+    {
+      key: 'totalMemory',
+      query: 'node_memory_MemTotal_bytes',
+    },
+    {
+      key: 'freeStorage',
+      query: 'node_filesystem_free_bytes',
+    },
+    {
+      key: 'totalStorage',
+      query: 'node_filesystem_avail_bytes',
+    },
+    {
+      key: 'cpu',
+      query: 'instance:node_cpu:rate:sum',
+    },
+    {
+      key: 'pods',
+      query: 'sum by(node)(kubelet_running_pod_count)',
+    },
+  ];
+  const promises = metrics.map(({ key, query }) => {
+    const url = `${PROMETHEUS_BASE_PATH}/api/v1/query?query=${query}`;
+    return coFetchJSON(url).then(({ data: { result } }) => {
```

Comment:
We want to control the polling. With `usePrometheusPoll` hook we cannot control the polling and we will have to use multiple instances of it since we require 6 metrices.